exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/statistics/Statistic.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.drill.exec.physical.impl.statistics;
+
+/*
+ * Base Statistics class - all statistics classes should extend this class
+ */
+public abstract class Statistic {
```

Comment:
@gparai we have an interface called Statistics which exists in the planner.index package.  While the packages are different, there can be confusion about the meaning of the Statistic class and the Statistics interface.  They are currently completely independent.  Would it even make sense to consolidate the two ?  One is reflecting the stats obtained from the storage plugin (including index stats) while the other is stats specifically collected by Drill.  If we maintain them as separate entities, then I suggest renaming this one to DrillStatistics.  If you have other thoughts including consolidation, let me know.